### ISSUE TITLE
missing file autogen-support.sh?

### ISSUE THREAD

**Issue by d-meiser** [rashack/guile-zeromq-3#2]
I'm trying to set up the build configuration using the `autogen.sh` script. But `autogen.sh` is trying to source `autogen-support.sh` which isn't in the repo. Is this file missing? How should I work around this? Thanks. Dominic

**Comment by rashack** [comment]
Sorry, I should have updated the documentation better. Look in HACKING, there you'll find the solution:
`autoreconf -i`...

**Comment by d-meiser** [comment]
Oh, cool thanks.

I ended up downloading `autogen.sh` from [here](http://buildconf.brlcad.org/) which also seems to work. But to be honest I don't even know what this script does. Not very familiar with autotools.

Btw. I took the liberty to  set up travis continuous integration for this library, mostly as an educational exercise for myself, see [https://github.com/d-meiser/guile-zeromq-3](https://github.com/d-meiser/guile-zeromq-3). Are you interested in any of those changes?
